src/index.js

```diff
@@ -10,17 +9,14 @@ export default class SplunkEvents {
     this.events = [];
     this.pendingEvents = [];
     this.isSendingEvents = false;
-
     this.endpoint = config.endpoint; // required
     this.token = config.token; // required
-
-    this.index = config.index; // optional
-    this.injectAditionalInfo = config.injectAditionalInfo !== undefined ? config.injectAditionalInfo : true;
+    this.account = config.account; // optional
+    this.injectAditionalInfo = config.injectAditionalInfo !== undefined ? config.injectAditionalInfo : false;
     this.autoFlush = config.autoFlush !== undefined ? config.autoFlush : true;
     this.autoRetryFlush = config.autoRetryFlush !== undefined ? config.autoRetryFlush : true;
-    this.source = config.source !== undefined ? config.source : 'datasource';
+    this.source = 'splunkeventsjs';
```

Comment:
I'm not sure, but I think that @Caldas uses this value to identify what regex must apply to extract the event data